style.css

```diff
@@ -0,0 +1,86 @@
+body{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+    background: #ff1d58;
+ 
+}
+
+.container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+
+}
+
+.container a{
+    position: relative;
+    width: 160px;
+    height: 60px;
+    background: white;
+    display: inline-block;
+    margin: 20px;
+}
```

Comment:
```js

.container a{

    position: relative;

    width: 160px;
    height: 60px;
    margin: 20px;

    background: white;
    display: inline-block;
} 
```

grouping means group the similar style property into multiple group by spacing . we done this for providing clear understanding to other devloper what we have done in styling if your not understand well so you can check this link which can remove your doubt :
https://css-tricks.com/different-logical-ways-group-css-properties/

Reply:
All styles are now grouped, kindly verify.

Reply:
All styles are now grouped, kindly verify.